.travis.yml

```diff
@@ -2,8 +2,7 @@ sudo: required
 
 language: node_js
 
-node_js:
-  - 11.14.0
+node_js: lts/*
```

Comment:
Don't you think using a non-fixed version may bring surprise?
IMO I could consider fixing it on a 12 to avoid an automatic switch when the next lts will be out.

Reply:
**(opinion)** I feel like it's a good idea to use the latest LTS for an open-source product where we don't know which version of Node our users  have installed. Common practices for OSS development is AFAIK to upgrade to latest version (minor/patch) when there is a problem.

It's better IMO to say :

> It does not work on your environment ? Please try to upgrade to latest LTS which may fix your problem. Our code is always tested on the latest LTS version.

Rather than: 

> It does not work on your environment ? Please try to **downgrade** to this particular version that our CI actually use.

-------

> Don't you think using a non-fixed version may bring surprise?

Yes, good surprise: Detecting bugs with last versions before users!

**It's an opinion, feel free to disagree!** Let me know what you think, and I will follow your recommendations! 🙏

Reply:
Very good, I was thinking I was on forestadmin-server repo.. sorry